optuna/study.py

```diff
@@ -228,7 +228,7 @@ def study_id(self):
         """
 
         message = (
-            'The use of `Study.study_id` is deprecated. ' 'Please use `Study.study_name` instead.'
+            "The use of `Study.study_id` is deprecated. " "Please use `Study.study_name` instead."
```

Comment:
I think these strings can be merged.

```suggestion
            "The use of `Study.study_id` is deprecated. Please use `Study.study_name` instead."
```